src/odemis/gui/comp/text.py

```diff
@@ -328,18 +332,21 @@ def _showDropDown(self, show=True):
         Either display the drop down list (show = True) or hide it (show = False).
         """
         if show:
-            size = self.dropdown.GetSize()
+            dwidth, dheight = self.dropdown.GetSize()
+            # Use the width of the text control
             width, height = self.GetSize()
+            width = max(width, LIST_MIN_WIDTH)
```

Comment:
Wouldn't it make more sense to add this check/line of code to line 99/100?
So:
>self._screen_size = (max(wx.SystemSettings.GetMetric(wx.SYS_SCREEN_X),LIST_MIN_WIDTH),
                                wx.SystemSettings.GetMetric(wx.SYS_SCREEN_Y))

Reply:
No, the screen size is only there to limit the position of the dropdown list, so that it's always fully within the screen limits if opening the dropdown very near the border.
The LIST_MIN_WIDTH is the minimum size of the widget. It's typically way smaller than the screen size (unless the user has a tiny tiny screen ;-) )